mock-test Fails during load (cannot find module sync-request)

### Environment
azure-pipelines-task-lib version: 2.9.3

### Issue Description
It looks like between 2.8.0 and 2.9.3, the mock test file was modified to import sync-request; however, in the package.json file this is still a dev-dependency and not a dependency.

### Expected behaviour
The package should load and mock tests should run.

### Actual behaviour
Mock runs fail with missing module 'sync-request'

### Steps to reproduce
1. Create a unit test that loads and utilizes mock-test from azure-pipelines-task-lib.
2. Execute unit test.

### Logs
> azure-devops-node-utilities-tests@3.0.0 mocha C:\Users\dckop\Documents\Source\TypeScript Pipeline Common Dev\TFS2018\tests
> mocha units/Common.Tests.js && mocha units/VersionFile.Tests.js

C:\Users\dckop\Documents\Source\TypeScript Pipeline Common Dev\TFS2018\tests\node_modules\yargs\yargs.js:1163
      else throw err
           ^

Error: Cannot find module 'sync-request'
    at Function.Module._resolveFilename (internal/modules/cjs/loader.js:636:15)
    at Function.Module._load (internal/modules/cjs/loader.js:562:25)
    at Module.require (internal/modules/cjs/loader.js:690:17)
    at require (internal/modules/cjs/helpers.js:25:18)
    at Object.<anonymous> (C:\Users\dckop\Documents\Source\TypeScript Pipeline Common Dev\TFS2018\tests\node_modules\azure-pipelines-task-lib\mock-test.js:10:19)
    at Module._compile (internal/modules/cjs/loader.js:776:30)
    at Object.Module._extensions..js (internal/modules/cjs/loader.js:787:10)
    at Module.load (internal/modules/cjs/loader.js:653:32)
    at tryModuleLoad (internal/modules/cjs/loader.js:593:12)
    at Function.Module._load (internal/modules/cjs/loader.js:585:3)
    at Module.require (internal/modules/cjs/loader.js:690:17)
    at require (internal/modules/cjs/helpers.js:25:18)
    at Object.<anonymous> (C:\Users\dckop\Documents\Source\TypeScript Pipeline Common Dev\TFS2018\tests\units\Common.Tests.js:12:26)
    at Module._compile (internal/modules/cjs/loader.js:776:30)
    at Object.Module._extensions..js (internal/modules/cjs/loader.js:787:10)
    at Module.load (internal/modules/cjs/loader.js:653:32)
    at tryModuleLoad (internal/modules/cjs/loader.js:593:12)
    at Function.Module._load (internal/modules/cjs/loader.js:585:3)
    at Module.require (internal/modules/cjs/loader.js:690:17)
    at require (internal/modules/cjs/helpers.js:25:18)
    at C:\Users\dckop\Documents\Source\TypeScript Pipeline Common Dev\TFS2018\tests\node_modules\mocha\lib\mocha.js:334:36
    at Array.forEach (<anonymous>)
    at Mocha.loadFiles (C:\Users\dckop\Documents\Source\TypeScript Pipeline Common Dev\TFS2018\tests\node_modules\mocha\lib\mocha.js:331:14)
    at Mocha.run (C:\Users\dckop\Documents\Source\TypeScript Pipeline Common Dev\TFS2018\tests\node_modules\mocha\lib\mocha.js:809:10)
    at Object.exports.singleRun (C:\Users\dckop\Documents\Source\TypeScript Pipeline Common Dev\TFS2018\tests\node_modules\mocha\lib\cli\run-helpers.js:108:16)
    at exports.runMocha (C:\Users\dckop\Documents\Source\TypeScript Pipeline Common Dev\TFS2018\tests\node_modules\mocha\lib\cli\run-helpers.js:142:13)
    at Object.exports.handler.argv [as handler] (C:\Users\dckop\Documents\Source\TypeScript Pipeline Common Dev\TFS2018\tests\node_modules\mocha\lib\cli\run.js:292:3)
    at Object.runCommand (C:\Users\dckop\Documents\Source\TypeScript Pipeline Common Dev\TFS2018\tests\node_modules\yargs\lib\command.js:242:26)
    at Object.parseArgs [as _parseArgs] (C:\Users\dckop\Documents\Source\TypeScript Pipeline Common Dev\TFS2018\tests\node_modules\yargs\yargs.js:1087:28)
    at Object.parse (C:\Users\dckop\Documents\Source\TypeScript Pipeline Common Dev\TFS2018\tests\node_modules\yargs\yargs.js:566:25)
    at Object.exports.main (C:\Users\dckop\Documents\Source\TypeScript Pipeline Common Dev\TFS2018\tests\node_modules\mocha\lib\cli\cli.js:68:6)
    at Object.<anonymous> (C:\Users\dckop\Documents\Source\TypeScript Pipeline Common Dev\TFS2018\tests\node_modules\mocha\bin\mocha:164:29)
    at Module._compile (internal/modules/cjs/loader.js:776:30)
    at Object.Module._extensions..js (internal/modules/cjs/loader.js:787:10)
    at Module.load (internal/modules/cjs/loader.js:653:32)
    at tryModuleLoad (internal/modules/cjs/loader.js:593:12)
    at Function.Module._load (internal/modules/cjs/loader.js:585:3)
    at Function.Module.runMain (internal/modules/cjs/loader.js:829:12)
    at startup (internal/bootstrap/node.js:283:19)
    at bootstrapNodeJSCore (internal/bootstrap/node.js:622:3)

--- a/node/package.json
+++ b/node/package.json
@@ -32,11 +32,11 @@
     "q": "^1.1.2",
     "semver": "^5.1.0",
     "shelljs": "^0.3.0",
-    "uuid": "^3.0.1"
+    "uuid": "^3.0.1",
+    "sync-request": "3.0.1"
   },
   "devDependencies": {
     "mocha": "5.2.0",
-    "sync-request": "3.0.1",
     "typescript": "^3.0.0"
   }
 }

